build_defs/souffle.bzl

```diff
@@ -100,6 +100,7 @@ def souffle_cc_library(
         srcs = [src],
         testonly = testonly,
         copts = [
+            "-fexceptions",
```

Comment:
What will happen internally? We will get rid of this flag?

Reply:
Actually, quite the opposite. We need `-fexceptions` internally, but it is the default in most C++ builds. So `-fexceptions` is usually a restatement of the default outside of Google, but will cause exceptions to be allowed inside of Google.

Reply:
SG.